### PR TITLE
#127 - Use testContainer

### LINKED ISSUE #68: NpmCommandsTest.java:178-181: transform method into...
The puzzle `64-5e3ded21` from #64 has to be resolved: 

https://github.com/artipie/npm-adapter/blob/430067da28fa7990de823767a6f4a88dccae5d97/src/test/java/com/artipie/npm/NpmCommandsTest.java#L178-L181

The puzzle was created by brastak on 21-May-20. 

Estimate: 90 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### LINKED ISSUE #127: Run NPM command in integration tests using Docker
Now integration tests in the project are running NPM command on local machine and so require NPM utility to be installed. To make it easier to run the tests NPM command should not be required. This can be archived by running it inside Docker container as it is done in Artipie integration tests and other adapters.

### PR DESCRIPTION
**Pull request by genryxy**
Closes #68 since this question is no longer relevant
Closes #127 
Run NPM command in test container instead of local machine

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmCommandsTest.java [hunk 1]
```diff
@@ -120,8 +136,8 @@ void tearDown() {
     @Test
     @Disabled
     void npmPublishWorks() throws Exception {
-        final Path project = new File("./src/test/resources/simple-npm-project/").toPath();
-        this.npmExecute("publish", project);
+        final String proj = "simple-npm-project";
+        this.exec("npm", "publish", proj, "--registry", this.url);
```
Comment (olegmoz) [inline src/test/java/com/artipie/npm/NpmCommandsTest.java:140]: @genryxy the test is disabled now, are you sure that changes to `project` variable value works?

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one question

**Comment by genryxy** [comment]
@olegmoz thanks, now it works . Usually with `Decode Failed to decode:Illegal character` as it is mentioned in the puzzle

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#131` is not assigned, can't get performer

**Pull request merged**